guide/voice/voice-connections.md

```diff
@@ -0,0 +1,117 @@
+
+It's important that you destroy voice connections when they are no longer required so that your bot leaves the voice channel, and to prevent memory leaks.
+
+```ts
+connection.destroy();
+```
+
+### Playing audio
+
+You can subscribe voice connections to audio players as soon as they're created. Audio players will try to stream audio to all their subscribed voice connections that are in the Ready state. Destroyed voice connections cannot be subscribed to audio players.
+
+```ts
+// Subscribe the connection to the audio player (will play audio on the voice connection)
+const subscription = connection.subscribe(audioPlayer);
+
+// subscription could be undefined if the connection is destroyed!
+if (subscription) {
+	// Unsubscribe after 5 seconds (stop playing audio on the voice connection)
+	setTimeout(() => subscription.unsubscribe(), 5_000);
+}
+```
+
+::: warning
+**Voice connections can be subscribed to one audio player at most.** If you try to subscribe to another audio player while already subscribed to one, the current audio player is unsubscribed and replaced with the new one.
+:::
+
+## Life cycle
+
+Voice connections have their own life cycle, with 5 distinct states. You can follow the methods discussed in the [life cycles](./life-cycles) section to subscribe to changes to voice connections.
+
+- **Signalling** - the initial state of a voice connection. The connection will be in this state when it is requesting permission to join a voice channel.
+
+- **Connecting** - the state a voice connection enters once it has permission to join a voice channel and is in the process of establishing a connection to it.
+
+- **Ready** - the state a voice connection enters once it has successfully established a connection to the voice channel. It is ready to play audio in this state.
+
+- **Disconnected** - the state a voice connection enters when the connection to a voice channel has been severed. This can occur even if the connection has not yet been established. You may choose to attempt to reconnect in this state.
+
+- **Destroyed** - the state a voice connection enters when it has been manually been destroyed. This will prevent it from accidentally being reused, and it will be removed from an in-memory tracker of voice connections.
+
+```ts
+const { VoiceConnectionStatus } = require('@discordjs/voice');
+
+connection.on(VoiceConnectionStatus.Ready, () => {
+	console.log('The connection has entered the Ready state - ready to play audio!');
+});
+```
+
+## Handling disconnects
+
+Disconnects can be quite complex to handle. There are 3 cases for handling disconnects:
+
+1. **Resumable disconnects** - this means there is no clear reason why the disconnect occurred. In this case, voice connections will automatically try to resume the existing session. The voice connection will enter the Connecting state. If this fails, it may enter a disconnected state again.
```

Comment:
```suggestion
1. **Resumable disconnects** - this means there is no clear reason why the disconnect occurred. In this case, voice connections will automatically try to resume the existing session. The voice connection will enter the `Connecting` state. If this fails, it may enter a `Disconnected` state again.
```